rust/decentraland-godot-lib/src/avatars/item.rs

```diff
@@ -83,49 +105,119 @@ impl DclWearableEntityDefinition {
 }
 
 #[godot_api]
-impl DclWearableEntityDefinition {
+impl DclItemEntityDefinition {
+    #[func]
+    fn get_emote_audio(&self, body_shape_id: String) -> GString {
+        let Some(representation) = self.get_emote_representation(&body_shape_id) else {
+            return GString::from("");
+        };
+
+        representation
+            .contents
+            .iter()
+            .find(|file_name| file_name.ends_with(".mp3") || file_name.ends_with(".ogg"))
+            .map(GString::from)
+            .unwrap_or_default()
+    }
+
+    #[func]
+    fn get_emote_loop(&self) -> bool {
+        let Some(emote_data) = self.inner.item.emote_data.as_ref() else {
+            return false;
+        };
+
+        emote_data.r#loop
+    }
+
+    /// Returns a 12 character long prefix for the emote id, removing the tokenId if it presents
```

Comment:
using 3 `///` usually is for a documentation comment, and I get warnings in the cargo run

we should start using double //